libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -251,297 +350,516 @@ class AC_PosControl
+    // Used to assess throttle margin and PID effectiveness.
     float get_vel_U_control_ratio() const { return constrain_float(_vel_u_control_ratio, 0.0f, 1.0f); }
 
-    /// crosstrack_error - returns horizontal error to the closest point to the current track
+    // Returns lateral distance to closest point on active trajectory in meters.
+    // Used to assess horizontal deviation from path.
     float crosstrack_error() const;
 
-    /// standby_NEU_reset - resets I terms and removes position error
-    ///     This function will let Loiter and Alt Hold continue to operate
-    ///     in the event that the flight controller is in control of the
-    ///     aircraft when in standby.
+    // Resets NEU position controller state to prevent transients when exiting standby.
+    // Zeros I-terms and aligns targets to current position.
     void standby_NEU_reset();
 
-    // get_measured_accel_U_cmss - returns the vertical (Up) acceleration in the earth frame, gravity-compensated, in cm/s/s (+ve = upward)
-    float get_measured_accel_U_cmss() const { return -(_ahrs.get_accel_ef().z + GRAVITY_MSS) * 100.0f; }
+    // Returns measured vertical (Up) acceleration in cm/s² (Earth frame, gravity-compensated).
+    // See get_measured_accel_U_mss() for full details.
+    float get_measured_accel_U_cmss() const { return get_measured_accel_U_mss() * 100.0; }
+
+    // Returns measured vertical (Up) acceleration in m/s² (Earth frame, gravity-compensated).
+    // Positive = upward acceleration.
+    float get_measured_accel_U_mss() const { return -(_ahrs.get_accel_ef().z + GRAVITY_MSS); }
 
-    /// returns true when the forward pitch demand is limited by the maximum allowed tilt
+    // Returns true if the requested forward pitch is limited by the configured tilt constraint.
     bool get_fwd_pitch_is_limited() const;
     
-    // set_disturb_pos_NE_cm - set the position disturbance in the north east plane
-    void set_disturb_pos_NE_cm(Vector2f disturb_pos) {_disturb_pos_ne_cm = disturb_pos;}
+    // Sets artificial NE position disturbance in centimeters.
+    // See set_disturb_pos_NE_m() for full details.
+    void set_disturb_pos_NE_cm(Vector2f disturb_pos_cm) { set_disturb_pos_NE_m(disturb_pos_cm * 0.01); }
+
+    // Sets artificial NE position disturbance in meters.
+    void set_disturb_pos_NE_m(Vector2f disturb_pos_m) { _disturb_pos_ne_m = disturb_pos_m; }
 
-    // set_disturb_vel_NE_cms - set the velocity disturbance in the north east plane
-    void set_disturb_vel_NE_cms(Vector2f disturb_vel) {_disturb_vel_ne_cms = disturb_vel;}
+    // Sets artificial NE velocity disturbance in cm/s.
+    // See set_disturb_vel_NE_ms() for full details.
+    void set_disturb_vel_NE_cms(Vector2f disturb_vel_cms) { set_disturb_vel_NE_ms(disturb_vel_cms * 0.01); }
+
+    // Sets artificial NE velocity disturbance in m/s.
+    void set_disturb_vel_NE_ms(Vector2f disturb_vel_ms) { _disturb_vel_ne_ms = disturb_vel_ms; }
```

Comment:
it's an existing issue so it doesn't need to be fixed but we could make the disturb_xxx argument all const references like this:

void set_disturb_pos_NE_cm(const Vector2f& disturb_pos_cm) { set_disturb_pos_NE_m(disturb_pos_cm * 0.01); }
void set_disturb_pos_NE_m(const Vector2f& disturb_pos_m) { _disturb_pos_ne_m = disturb_pos_m; }
void set_disturb_vel_NE_cms(const Vector2f& disturb_vel_cms) { set_disturb_vel_NE_ms(disturb_vel_cms * 0.01); }
void set_disturb_vel_NE_ms(const Vector2f& disturb_vel_ms) { _disturb_vel_ne_ms = disturb_vel_ms; }